criu/include/mman.h

```diff
@@ -13,5 +13,8 @@
 #ifndef MADV_DONTDUMP
 #define MADV_DONTDUMP 16
 #endif
+#ifndef MADV_GUARD_INSTALL
+#define MADV_GUARD_INSTALL 102
+#endif
```

Comment:
This should be defined before it's used in restorer ;-)

Reply:
ah, I use `MADV_GUARD_INSTALL` in kdat and restorer code, so I just moved this commit before others to have a right ordering and to make it possible to do `git bisect` without any problems.

Or do you think it's better to define `MADV_GUARD_INSTALL` right in `restorer.c` too?

Reply:
I meant to move the commit ahead, yeah